app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !session[:id].nil?
```

Comment:
good chance to use our new `logged_in?` method, and for any other `session[:id]` checks in your application.